shared/reducers/chat.js

```diff
@@ -532,13 +531,22 @@ function reducer(state: Constants.State = initialState, action: Constants.Action
       )
     }
     case 'chat:addPendingConversation': {
-      const {participants} = action.payload
+      const {participants, temporary} = action.payload
       const sorted = participants.sort()
       const conversationIDKey = Constants.pendingConversationIDKey(sorted.join(','))
-      return state.set(
-        'pendingConversations',
-        state.get('pendingConversations').set(conversationIDKey, List(sorted))
-      )
+      return state
+        .set('pendingConversations', state.get('pendingConversations').set(conversationIDKey, List(sorted)))
+        .update('tempPendingConversations', tempPendingConversation =>
```

Comment:
:sheep: Maybe `tempPendingConversation` => `tempPendingConversations`? I don't mind it if this was intended, just pointing it out if not.